packages/netlify-cms-widget-markdown/src/MarkdownControl/index.js

```diff
@@ -32,16 +34,23 @@ export default class MarkdownControl extends React.Component {
     super(props);
     editorControl = props.editorControl;
     _getEditorComponents = props.getEditorComponents;
-    this.state = { mode: localStorage.getItem(MODE_STORAGE_KEY) || 'visual' };
+    this.state = {
+      mode: localStorage.getItem(MODE_STORAGE_KEY) || 'visual',
+      pendingFocus: false,
```

Comment:
`pendingFocus` allows us to return focus to the newly instantiated editor after the Rich Text/Markdown toggle is activated.